tests/AppTest.php

```diff
@@ -4,9 +4,13 @@
 use Illuminate\Foundation\Testing\DatabaseMigrations;
 use Illuminate\Foundation\Testing\DatabaseTransactions;
 use Otp\Otp;
+use App\User;
 
-class ExampleTest extends TestCase
+class AppTest extends TestCase
 {
+    use DatabaseMigrations;
```

Comment:
This will migrate your database each time you run a test. DatabaseTransactions should be enough. Your test database should be migrated before running the tests.